java/ql/test/kotlin/library-tests/parameter-defaults/PrintAst.expected

```diff
@@ -0,0 +1,849 @@
+#   82|         0: [SuperConstructorInvocationStmt] super(...)
+#   82|         1: [BlockStmt] { ... }
+#   84|     2: [Method] enclosing
+#   84|       3: [TypeAccess] Unit
+#   84|       5: [BlockStmt] { ... }
+#   86|         0: [LocalTypeDeclStmt] class ...
+#   86|           0: [LocalClass] 
+#   86|             1: [Constructor] 
+#   86|               5: [BlockStmt] { ... }
+#   86|                 0: [SuperConstructorInvocationStmt] super(...)
+#   86|             2: [Method] f
+#   86|               3: [TypeAccess] Unit
+#-----|               4: (Parameters)
+#   86|                 0: [Parameter] x
+#   86|                   0: [TypeAccess] String
+#   86|                 1: [Parameter] y
+#   86|                   0: [TypeAccess] String
+#   86|                 2: [Parameter] z
+#   86|                   0: [TypeAccess] String
+#   86|               5: [BlockStmt] { ... }
+#   87|                 0: [ExprStmt] <Expr>;
+#   87|                   0: [MethodAccess] sink(...)
+#   87|                     -1: [TypeAccess] TestKt
+#   87|                     0: [VarAccess] y
+#   86|             3: [Method] f$default
+#   86|               3: [TypeAccess] Unit
+#-----|               4: (Parameters)
+#   86|                 0: [Parameter] p0
+#   86|                   0: [TypeAccess] String
+#   86|                 1: [Parameter] p1
+#   86|                   0: [TypeAccess] String
+#   86|                 2: [Parameter] p2
+#   86|                   0: [TypeAccess] String
+#   86|                 3: [Parameter] p3
+#   86|                   0: [TypeAccess] int
+#   86|                 4: [Parameter] p4
+#   86|                   0: [TypeAccess] Object
+#   86|               5: [BlockStmt] { ... }
+#   86|                 0: [IfStmt] if (...)
+#   86|                   0: [EQExpr] ... == ...
+#   86|                     0: [AndBitwiseExpr] ... & ...
+#   86|                       0: [IntegerLiteral] 2
+#   86|                       1: [VarAccess] p3
+#   86|                     1: [IntegerLiteral] 0
+#   86|                   1: [ExprStmt] <Expr>;
+#   86|                     0: [AssignExpr] ...=...
+#   86|                       0: [VarAccess] p1
+#   86|                       1: [VarAccess] p0
+#   86|                 1: [IfStmt] if (...)
+#   86|                   0: [EQExpr] ... == ...
+#   86|                     0: [AndBitwiseExpr] ... & ...
+#   86|                       0: [IntegerLiteral] 4
+#   86|                       1: [VarAccess] p3
+#   86|                     1: [IntegerLiteral] 0
+#   86|                   1: [ExprStmt] <Expr>;
+#   86|                     0: [AssignExpr] ...=...
+#   86|                       0: [VarAccess] p2
+#   86|                       1: [StringLiteral] hello world
+#   86|                 2: [ReturnStmt] return ...
+#   86|                   0: [MethodAccess] f(...)
+#   86|                     -1: [ClassInstanceExpr] new (...)
+#   86|                       -3: [TypeAccess] Object
```

Comment:
I don't think we should create an instance here, the instance is created at [the call site](https://github.com/github/codeql/pull/10683/commits/7e1a6035449e99cedea7fcb0f3e861a89ac4852c#diff-3e5daacdf0970b183314af74d106e324d2530bdb765f4aecf5c524518efd4757R571-R572), and should be propagated here as the qualifier.